src/ExposureHistory/History/index.tsx

```diff
@@ -34,11 +36,26 @@ const History: FunctionComponent<HistoryProps> = ({
 }) => {
   const { t } = useTranslation()
   const navigation = useNavigation()
+  const [refreshing, setRefreshing] = useState(false)
+  const prevExposuresRef = useRef<ExposureDatum[]>()
+
+  useEffect(() => {
+    prevExposuresRef.current = exposures
+  })
 
   const handleOnPressMoreInfo = () => {
     navigation.navigate(Screens.MoreInfo)
   }
 
+  const handleRefresh = () => {
```

Comment:
Thoughts on this for consistency?

```suggestion
  const handleOnRefresh = () => {
```